brion/plugin/compartmentReportHDF5.cpp

```diff
@@ -595,6 +598,40 @@ H5::DataSet& CompartmentReportHDF5::_getDataset( const uint32_t gid )
     return it->second;
 }
 
+void CompartmentReportHDF5::_createMetaData()
+{
+    H5::Group root = _file.openGroup( "/" );
+    const std::string creatorName = "Brion";
+    const H5::StrType creatorType( H5::PredType::C_S1, creatorName.length( ));
+    H5::Attribute creatorAttr = root.createAttribute( "creator", creatorType,
+                                                       H5S_SCALAR );
+    creatorAttr.write( creatorType, creatorName );
```

Comment:
The last three statements are almost copy-pasted below and also used in the morphology v2 writer. Add a function instead of copy-paste?